### ISSUE TITLE
gh does not work with fish

### ISSUE THREAD

**Issue by sbarratt** [dickeyxxx/gh#22]
`fisher dickeyxxx/gh` does not work.

```
Unsupported use of '='. In fish, please use 'set GH_BASE_DIR $GH_BASE_DIR:-$HOME/src'.
~/.config/fish/functions/bb.fish (line 1): GH_BASE_DIR=${GH_BASE_DIR:-$HOME/src}
                                           ^
from sourcing file ~/.config/fish/functions/bb.fish
	called on line 767 of file ~/.config/fish/functions/fisher.fish

in function '__fisher_plugin_enable'
	called on line 639 of file ~/.config/fish/functions/fisher.fish
	with parameter list '/Users/shane/.config/fisherman/gh'

in function '__fisher_update'
	called on line 257 of file ~/.config/fish/functions/fisher.fish
	with parameter list 'dickeyxxx/gh'

in function 'fisher'
	called on standard input
	with parameter list 'update dickeyxxx/gh'

Unsupported use of '='. In fish, please use 'set GH_BASE_DIR $GH_BASE_DIR:-$HOME/src'.
~/.config/fish/functions/gh.fish (line 1): GH_BASE_DIR=${GH_BASE_DIR:-$HOME/src}
                                           ^
from sourcing file ~/.config/fish/functions/gh.fish
	called on line 767 of file ~/.config/fish/functions/fisher.fish

in function '__fisher_plugin_enable'
	called on line 639 of file ~/.config/fish/functions/fisher.fish
	with parameter list '/Users/shane/.config/fisherman/gh'

in function '__fisher_update'
	called on line 257 of file ~/.config/fish/functions/fisher.fish
	with parameter list 'dickeyxxx/gh'

in function 'fisher'
	called on standard input
	with parameter list 'update dickeyxxx/gh'

Unsupported use of '='. In fish, please use 'set GH_BASE_DIR $GH_BASE_DIR:-$HOME/src'.
~/.config/fish/functions/gl.fish (line 1): GH_BASE_DIR=${GH_BASE_DIR:-$HOME/src}
                                           ^
from sourcing file ~/.config/fish/functions/gl.fish
	called on line 767 of file ~/.config/fish/functions/fisher.fish

in function '__fisher_plugin_enable'
	called on line 639 of file ~/.config/fish/functions/fisher.fish
	with parameter list '/Users/shane/.config/fisherman/gh'

in function '__fisher_update'
	called on line 257 of file ~/.config/fish/functions/fisher.fish
	with parameter list 'dickeyxxx/gh'

in function 'fisher'
	called on standard input
	with parameter list 'update dickeyxxx/gh'

Unsupported use of '='. In fish, please use 'set GH_BASE_DIR $GH_BASE_DIR:-$HOME/src'.
~/.config/fish/completions/gh.fish (line 1): GH_BASE_DIR=${GH_BASE_DIR:-$HOME/src}
                                             ^
from sourcing file ~/.config/fish/completions/gh.fish
	called on line 801 of file ~/.config/fish/functions/fisher.fish

in function '__fisher_plugin_enable'
	called on line 639 of file ~/.config/fish/functions/fisher.fish
	with parameter list '/Users/shane/.config/fisherman/gh'

in function '__fisher_update'
	called on line 257 of file ~/.config/fish/functions/fisher.fish
	with parameter list 'dickeyxxx/gh'

in function 'fisher'
	called on standard input
	with parameter list 'update dickeyxxx/gh'
```

**Comment by patrick96** [comment]
This is my bad. Sorry!
I just assumed variable assignment works the same as in `sh`, gotta do some testing the next time
Is this fixed with #23?

**Comment by dickeyxxx** [comment]
it's alright, I should've tested it before merging
